content/en/getting_started/integrations/google_cloud.md

```diff
@@ -0,0 +1,351 @@
+    - link: 'https://www.datadoghq.com/blog/gke-dashboards-integration-improvements/'
+      tag: 'Blog'
+      text: 'New GKE dashboards and metrics provide deeper visibility into your environment'
+    - link: 'https://www.datadoghq.com/blog/google-cloud-private-service-connect/'
+      tag: 'Blog'
+      text: 'Access Datadog privately and monitor your Google Cloud Private Service Connect usage'
+    - link: 'https://www.datadoghq.com/blog/track-bigquery-costs-performance/'
+      tag: 'Blog'
+      text: 'Monitor BigQuery with Datadog'
+    - link: 'https://www.datadoghq.com/blog/google-cloud-cost-management/'
+      tag: 'Blog'
+      text: 'Empower engineers to take ownership of Google Cloud costs with Datadog'
+    - link: 'https://www.datadoghq.com/blog/collect-traces-logs-from-cloud-run-with-datadog/'
+      text: 'Collect traces, logs, and custom metrics from your Google Cloud Run services with Datadog'
+      tag: 'Blog'
+---
+
+## Overview
+
+Integrating your Google Cloud projects with Datadog allows you to monitor your Google Cloud services within the Datadog platform.
+This guide provides an overview of how to get started with the Datadog Google Cloud integration.
+
+## Prerequisites
+1) Create a [Datadog account][1]
+2) Set up a [Service Account][2] in any of your Google Cloud projects
+3) Review these Google Cloud Prerequisites:
+
+{{% site-region region="us,us3,us5,eu,ap1" %}}
+- If your organization restricts identities by domain, you must add Datadog's customer identity `C0147pk0i` as an allowed value in your policy.
+{{% /site-region %}}
+- Datadog does not support [scoping projects][42] that pull in metrics from multiple other projects.
+- The Google Cloud integration requires the below APIs to be enabled **for each of the projects** you want to monitor:
+
+[Cloud Monitoring API][7] 
+: Allows Datadog to query your Google Cloud metric data.
+
+[Compute Engine API][8] 
+: Allows Datadog to discover compute instance data.
+
+[Cloud Asset API][9]
+: Allows Datadog to request Google Cloud resources and link relevant labels to metrics as tags.
+
+[Cloud Resource Manager API][10] 
+: Allows Datadog to append metrics with the correct resources and tags.
+
+[IAM API][11]
+: Allows Datadog to authenticate with Google Cloud.
+
+[Google Cloud Billing API][12] 
+: Enables accurate billing for API calls.
+
+<div class="alert alert-info">You can confirm if these APIs are enabled by heading to <a href="https://console.cloud.google.com/apis/dashboard"><b>Enabled APIs & Services</a></b> under each project in your Google Cloud workspace.</div>
+
+## Setup 
+
+For the most comprehensive level of monitoring in your Google Cloud environment, integrate Datadog with [Google Cloud][4] using [service account impersonation][5] at the organization or folder level.
+
+After reviewing the above [Prerequisites][3], follow these step-by-step instructions to set up the Google Cloud integration:
```

Comment:
why do we need this?

Reply:
Sorry do you mean this section or this sentence?